### DISCUSSION TITLE
PK-Sim and Ketoconazole model

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1339] [Questions & Problems]
Is there no ketoconazole model file in pk-sim's model library?

**Comment by StephanSchaller** [comment]
Currently in development. Publication soon forthcoming

**Reply by Fatima-Marok** [reply]
Exactly, it's still under development. We plan it to be published by the end of the year.

**Reply by WTY-tt** [reply]
All right, looking forward to the final model.
